.github/workflows/web_tests.yml

```diff
@@ -27,8 +30,7 @@ jobs:
       working-directory: cmd/fyne_demo
       run: go build
 
-    - name: Build GopherJS and Wasm full website
```

Comment:
Would this not be part of the GopherJS change?

Reply:
No. The tests would constantly fail without that test removed

Reply:
Because of the version change? We could set GOPHERJS_GOROOT and resume them working can't we? https://developer.fyne.io/started/webapp

Reply:
Yeah. Is that really worth the effort if we're about to rip out GopherJS very soon?

Reply:
Makes sense to have a single PR that can be reverted yes. I'll try to revert the action to use 1.17 for now.

Reply:
If it turns out to be a pain just say so, don't want to add much effort for the what-if cases, but i appreciate you looking at it.

Reply:
I have tried the simple solution of reverting all changes to the web test and downgraded fredbi/uri as it didn't support Go 1.17. We can upgrade later on when other Go 1.19 changes land